sources/api/apiserver/src/server/mod.rs

```diff
@@ -352,6 +354,27 @@ async fn get_configuration_files(
     Ok(ConfigurationFilesResponse(resp))
 }
 
+/// Reboots the machine
+async fn reboot() -> Result<HttpResponse> {
+    debug!("Rebooting now");
+    let output = Command::new("/sbin/shutdown")
+        .arg("-r")
+        .arg("now")
+        .output()
+        .context(error::Shutdown)?;
+    ensure!(
+        output.status.success(),
+        error::Reboot {
+            exit_code: match output.status.code() {
+                Some(code) => code,
+                None => output.status.signal().unwrap_or_else(|| 1),
```

Comment:
```suggestion
                None => output.status.signal().unwrap_or(1),
```
Shouldn't need the closure for a Copy type like an integer that's trivial to construct.